cmake/roboschool.cmake

```diff
@@ -66,22 +72,24 @@ set(BULLET_LIBRARIES
   "${INSTALL_DIR}/lib/libBulletInverseDynamics.so"
   "${INSTALL_DIR}/lib/libPhysicsClientC_API.so")
 
-set(ROBOSCHOOL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/external/roboschool)
-
 ## roboschool
 ExternalProject_Add(roboschool
+  PREFIX ${EXTERNAL_PROJECT_PREFIX}
   GIT_REPOSITORY "https://github.com/skylian/roboschool"
   GIT_TAG "remove_qt"
-  SOURCE_DIR ${ROBOSCHOOL_DIR}
+  BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/external/roboschool"
+  INSTALL_DIR "${EXTERNAL_PROJECT_PREFIX}/roboschool"
   DEPENDS assimp bullet glm opencv
   LOG_DOWNLOAD ON
   BUILD_COMMAND ""
   INSTALL_COMMAND ""
   CONFIGURE_COMMAND ""
 )
 
+ExternalProject_Get_Property(roboschool SOURCE_DIR)
+set(ROBOSCHOOL_DIR ${SOURCE_DIR})
 ## make roboschool
-set(ROBOSCHOOL_INCLUDE_DEPS ${ASSIMP_INCLUDE_PATH} ${GLM_INCLUDE_PATH} ${BULLET_INCLUDE_PATHS} ${OPENCV_INCLUDE_PATH})
+set(ROBOSCHOOL_INCLUDE_DEPS ${ASSIMP_INCLUDE_PATH} ${GLM_INCLUDE_PATH} ${BULLET_INCLUDE_PATHS} ${OpenCV_INCLUDE_DIRS})
 set(ROBOSCHOOL_LIB_DEPS ${ASSIMP_LIBRARIES} ${BULLET_LIBRARIES} ${OPENCV_LIBS})
```

Comment:
Where do you use ROBOSCHOOL_DIR and why change opencv_include_path to opencv_include_dirs?